modules/virt-editing-vm-web.adoc

```diff
@@ -14,29 +14,28 @@
 ifeval::["{context}" == "virt-editing-vm-template"]
 :virt-vm-template:
 :object: virtual machine template
-:object-gui: Virtual Machine Template
+:object-gui: Virtual Machines
 endif::[]
 
 ifeval::["{context}" == "virt-edit-vms"]
 :virt-vm:
 :object: virtual machine
-:object-gui: Virtual Machine
+:object-gui: Virtual Machines
 endif::[]
 
 [id="virt-editing-vm-web_{context}"]
 
 = Editing a {object} in the web console
 
-Edit select values of a {object} in the *{object-gui} Overview* screen
-of the web console by clicking on the pencil icon next to the relevant field.
-Other values can be edited using the CLI.
+Edit select values of a {object} that displays in the *Virtual Machines Overview* screen of the web console by clicking on the pencil icon next to the relevant field. Other values can be edited using the CLI.
+
+Labels and annotations are editable for both preconfigured Red Hat templates and your custom virtual machine templates. All other values are editable only for custom virtual machine templates that users have created using the Red Hat templates or the interactive wizard.
 
 .Procedure
 
 . Click *Workloads* -> *Virtualization* from the side menu.
-. Click the *{object-gui}s* tab.
-. Select a {object} to open the *{object-gui} Overview* screen.
-. Click the *Details* tab.
+. Click the *{object-gui}* tab.
+. Select a {object} to open the *Virtual Machine Overview* tab.
```

Comment:
Change tab to screen to be consistent with the rest of our docs?

Reply:
Edit was made.

Reply:
Should step 2 be tab and step 3 screen? I am not sure how the conditionals are working here so I could be wrong.